#!/usr/bin/env python
PACKAGE="screen_grab"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("x_offset", int_t, 1, "x offset", 0, 0, 2048)
gen.add("y_offset", int_t, 1, "y offset", 0, 0, 2048)
gen.add("width", int_t, 1, "width", 100, 0, 2048)
gen.add("height", int_t, 1, "height", 100, 0, 2048)
gen.add("update_rate", double_t, 2, "updates per second", 5, 0, 50)

exit(gen.generate(PACKAGE, "screen_grab", "ScreenGrab"))
